DiscordBot/Program.cs

```diff
@@ -24,9 +25,18 @@ private static async Task MainAsync()
                 LogLevel = LogLevel.Debug
             });
 
+            DependencyCollection dependencies;
```

Comment:
This should be declared when you're assigning to it. Avoiding splitting up declarations and assignments whenever possible! 🙂